README.md

```diff
@@ -4,6 +4,11 @@
 
 FastSeq provides efficient implementation of popular sequence models (e.g. [Bart](https://arxiv.org/pdf/1910.13461.pdf), [ProphetNet](https://github.com/microsoft/ProphetNet)) for text generation, summarization, translation tasks etc. It automatically optimizes inference speed based on pupular NLP toolkits (e.g. [FairSeq](https://github.com/pytorch/fairseq) and [HuggingFace-Transformers](https://github.com/huggingface/transformers)) without accuracy loss. All these can be easily done (no need to change any code/model/data if using our command line tool, or simply add one-line code `import fastseq` if using source code).
 
+## Features:
+* EL-Attention: Memory Efficient Lossless Attention for Generation ([Yan et al., 2021](https://arxiv.org/abs/2105.04779))
+* GPU-based Block N-Gram Repeats
+* Asynchronous Pipeline for Postprocess
+
```

Comment:
Add the optimized cache in the attention?

Reply:
EL-Attention covered all of our attention opt.